assets/css/appStyles.scss

```diff
@@ -74,6 +74,10 @@ div.MuiGrid-root.MuiGrid-item .govuk-summary-list:last-of-type ~ br {
   border-radius: 50%;
 }
 
+ul.govuk-list.govuk-list--bullet li > span:nth-child(2) {
```

Comment:
I believe this will be a risk to override style on generic classes.
Is there a scope to use **class="govuk-!-margin-left-0"** on span where we want to override style

Reply:
Hey Akshay, Apologies. I meant to make this a draft (for that exact reason). It's raw html from Pega that needs changing. This is more of a emergency option if needed.